packages-exp/analytics-exp/src/index.ts

```diff
@@ -80,6 +78,18 @@ function registerAnalytics(): void {
   }
 }
 
-export { factory, resetGlobalVars, getGlobalVars };
-
 registerAnalytics();
+
+export * from './api';
+
+export {
+  DataLayer,
```

Comment:
What is the use case for DataLayer? It feels like an internal type that's only useful for us.

Reply:
You're right, removed.